<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="programmGroup">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>341</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Программа</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="CodeEditor" name="programmText"/>
      </item>
      <item>
       <widget class="QPushButton" name="runButtton">
        <property name="text">
         <string>Запуск</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="registersGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Регистры</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="registersLayout">
      <item>
       <widget class="QTableWidget" name="registersTable"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="memoryGroup">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>341</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Память</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>321</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="memoryLayout">
      <item>
       <widget class="QTableWidget" name="memoryTable"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
